### ISSUE TITLE
Formatter/Validator for HTML

### ISSUE THREAD

**Issue by exoego** [fomantic/Fomantic-UI-Docs#152]
Writing valid HTML and format are sometimes forgetten and then spend energy to fix them like doing so in https://github.com/fomantic/Fomantic-UI-Docs/pull/151. 

I think automatic and uniform way of validating/format saves our energy for sustaining docs.
Let's consider to introduce something like https://prettier.io/ into CI.

**Comment by hammy2899** [comment]
Agreed, we might want to look at alternatives to prettier because it will complain about probably 90% of the codebase and that will be a lot of work to change.
